ci_framework/hooks/playbooks/config_ceph_backend_cinder_glance.yml

```diff
@@ -0,0 +1,86 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - ansible.builtin.find:
+        paths: "{{ cifmw_basedir }}/artifacts/manifests/{{ cifmw_install_yamls_defaults['NAMESPACE'] }}/openstack/cr/"
+        patterns: "*.yaml"
+        excludes: "kustomization.yaml"
+      register: find_result
+
+    - name: Set CR path fact
+      ansible.builtin.set_fact:
+        cifmw_edpm_kustomize_cr_path: "{{ find_result.files|map(attribute='path') | list | first }}"
+
+    - name: Kustomize Cinder configuration
+      vars:
+        cifmw_edpm_kustomize_content: |-
```

Comment:
We will need to change how `cifmw_edpm_kustomize_content` is managed, as for now, we only support one kustomization. We will need to convert this variable to a list